client/components/data/purchases/site-purchases/index.jsx

```diff
@@ -1,65 +0,0 @@
-/**
- * External dependencies
- */
-import React from 'react';
-
-/**
- * Internal dependencies
- */
-import { fetchSitePurchases } from 'lib/upgrades/actions';
-import PurchasesStore from 'lib/purchases/store';
-import StoreConnection from 'components/data/store-connection';
-import sitesFactory from 'lib/sites-list';
-
-/**
- * Module variables
- */
-const sites = sitesFactory(),
-	stores = [
-		PurchasesStore,
-		sites
```

Comment:
Sorry Drew, I really tried but we can't connect site to the global store yet. The main reason is that the site filled in the `sites-list` have custom attributes such as `fetchingSettings` which are not duplicated in the store (which contains raw data from the API). `fetchingSettings` is used at many places in the SiteSettings forms: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/form-general.jsx#L48
So this change would require `siteSettings` to be moved to the global store first.